### PR TITLE
Small updates on Language sub-directories

### PR DESCRIPTION
**Pull request by Craig-Elre**

### CODE REVIEW

File: docs/src/rsh/appinit/index.md [hunk 1]
```diff
@@ -114,6 +121,9 @@ Each function must occur exactly once in the entire program.
 
 ### {#ref-programs-appinit-view} View Definition
 
+Views generate a funtion that anyone can call to learn the state without being tied to the implementation.
```
Comment (jeapostrophe) [inline docs/src/rsh/appinit/index.md:124]: Typoe

File: docs/src/rsh/appinit/index.md [hunk 2]
```diff
@@ -114,6 +121,9 @@ Each function must occur exactly once in the entire program.
...
+On Ethereum and Conflux, `view` can call contract-to-contract and API functions that are guaranteed to be read-only.
```
Comment (jeapostrophe) [inline docs/src/rsh/appinit/index.md:125]: I don't know what you're getting at here.

File: docs/src/rsh/step/index.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ Alice.only(() => {
   const pretzel = interact.random(); });
 ```
 
-A local step statement is written `{!rsh} PART.only(() => BLOCK)`, where `{!rsh} PART` is a participant identifier and `{!rsh} BLOCK` is a block.
+A local step statement is written `{!rsh} PART.only(() => BLOCK)`, where `{!rsh} PART` is a participant identifier and `{!rsh} BLOCK` is a <a class="ui-tooltip" title="a block is a grouping of zero or more statements inside a single statement"><span style="cursor: help;">block</span></a>.
```
Comment (jeapostrophe) [inline docs/src/rsh/step/index.md:19]: You should make a Jira card to add this as a feature in the docs generator

File: docs/src/rsh/step/index.md [hunk 4]
```diff
@@ -56,7 +57,7 @@ each([Alice, Bob], () => {
   const pretzel = interact.random(); });
 ```
 
-An @{defn("each")} local step statement can be written as `{!rsh} each(PART_TUPLE () => BLOCK)`, where `{!rsh} PART_TUPLE` is a tuple of participants and `{!rsh} BLOCK` is a block.
+An @{defn("each")} local step statement can be written as `{!rsh} each(PART_TUPLE () => BLOCK)`, where `{!rsh} PART_TUPLE` is a <a class="ui-tooltip" title="variables that are used to store multiple items"><span style="cursor: help;">tuple</span></a> of participants and `{!rsh} BLOCK` is a block.
```
Comment (jeapostrophe) [inline docs/src/rsh/step/index.md:60]: Tuples are not variables, they are values. You could say "values that store multiple other values"

File: docs/src/tool/index.md [hunk 5]
```diff
@@ -60,28 +60,27 @@ Tip: try entering your desired year in the filter box to skip other tag types.
...
-If no `EXPORT` is provided, then all the exported Reach.Apps will be compiled. If there are no
-Reach.Apps exported, then the program will be compiled as a library, where its exports are available
-to other Reach programs and frontends. The output name of a library is the same as if it exported a Reach.App
-named `default`.
+If no `EXPORT` is provided, then all the exported `{!rsh} Reach.App`s will be compiled.
+If there are no `{!rsh} Reach.App`s exported, then the program will be compiled as a library, where its exports are available to other Reach programs and frontends.
+The output name of a library is the same as if it exported a `{!rsh} Reach.App` named `default`.
 
-`reach compile` supports the following options:
+`@{ref("cmd", "compile")}reach compile` supports the following options:
```
Comment (jeapostrophe) [inline docs/src/tool/index.md:83]: This ref should be outside the "`"

File: docs/src/tool/index.md [hunk 6]
```diff
@@ -105,18 +104,24 @@ named `default`.
 + The environment variable `REACH_DEBUG`, if set to any non-empty value, enables debug messages from the Reach compiler, which will appear in the console.
   This debug information includes: the estimated cost of the contract on Algorand.
   This variable automatically enabled `--intermediate-files`.
+  
+A `reach compile` usage example is available in [Overview](##over-compile).
 
-## {#ref-usage-init} `reach init`
+# {#ref-usage-init} `reach init`
```
Comment (jeapostrophe) [inline docs/src/tool/index.md:110]: Weren't these changes made before? I think you need to rebase

**Review by jeapostrophe — CHANGES_REQUESTED** [review]

File: docs/src/rsh/appinit/index.md (hunk 2 above)
Comment (Craig-Elre) [reply docs/src/rsh/appinit/index.md:125]: I wrote that based on information I found in our documentation and online, but I tried to search for it on our documentation, and I am not finding it. If it doesn't make sense to you, I think that is a good clue that I missed the mark and we should not include it. I will remove it.

Comment (jeapostrophe) [reply docs/src/rsh/appinit/index.md:125]: It is true that they are part of the on-chain API and that they are read-only, but the phrasing of this is confusing to me.

File: docs/src/tool/index.md (hunk 6 above)
Comment (jeapostrophe) [reply docs/src/tool/index.md:110]: You should rebase this and I believe it will get fixed

Comment (Craig-Elre) [reply docs/src/tool/index.md:110]: I tried to rebase to master, but it is giving me the invalid paths from before, I tried to do the --reapply-cherry-picks, and it didn't work either.

**Pull request closed without merging**